Web_Development/FrontEnd/Colors-In-CSS_by_RB/Playing with Colors.md

```diff
@@ -0,0 +1,469 @@
+All components must be specified using the same number of digits.A color in hexadecimal string notation always begins with the character "#". 
+After that come the hexadecimal digits of the color code. The string is case-insensitive. **"#rrggbb"**
+
+In CSS, HSL functional notation is that, a color can be specified using hue, saturation, and lightness (HSL) in the form:
+
+**hsl(hue, saturation, lightness)**
+
+Hue is a degree on the color wheel from 0 to 360. 0 is red, 120 is green, and 240 is blue.
+Saturation is a percentage value, 0% means a shade of gray, and 100% is the full color.
+Lightness is also a percentage, 0% is black, 50% is neither light or dark, 100% is white.
+Designers and artists often prefer to work using the HSL (Hue/Saturation/Luminosity) color method. 
+
+The hsl( )CSS function is very similar to the rgb() function in usage.  
+  
+## 2. The ways of making colors translucent in CSS
+  
+To make the desired color into a lighter form we can use “Opacity” as a property for the elements.
+The opacity property specifies the opacity/transparency of an element. It can take a
+value from 0.0 - 1.0. The lower value, the more transparent:  
+  
+
+![](images/Screenshot%202021-09-29%20091241.jpg " Picture") 
+  
+  (PC. from w3school.com) [Reference link](https://www.w3schools.com/css/css_background.asp)
+  
+We use RGBA model instead of RGB functional notation model and it is also included a fourth component: the alpha channel (or opacity).
+The alpha channel is a number between 0.0 (fully transparent) and 1.0 (fully opaque). We can also specify a percentage where 0% is the same as 0.0 and 100% is the same as 1.0.
+For example, a bright red that's 50% opaque can be represented as rgb(255, 0, 0, 0.5)or rgb(100%, 0, 0, 50%).
+
+**In HEX string notation model, we use it as "#rrggbbaa" OR "#rgba" for opacity property.** 
+Specifies a color whose red component is the hexadecimal number 0xrr, green component is 0xgg, and blue component is 0xbb. The alpha channel is specified by0xaa; the lower this value is, the more translucent the color becomes.
+
+As an example, you can represent the opaque color bright blue as "#0000ff" or "#00f". To make it 25% opaque, one can use "#0000ff44" or "#00f4". 
+  
+We can also define “opacity” property for an element in CSS like as following:
+
+Eg.
+
+```  
+.
+.
+.
+ div {
+
+background-color: green;
+opacity: 0.6;
+  
+ }
+ 
+ .
+ .
+ . 
+```
+ Output:
+  
+ 
+![](images/Screenshot%202021-09-29%200912411.jpg " Picture")
+ 
+  
+  
+## 3. The ways of adding colors in CSS
+•	**External CSS:** Specifying colors in stylesheets. External styles are defined within the `<link>` element, inside the `<head>` section of an HTML file.
+
+•	**Internal CSS:** Adding all the required properties via `<style>` tag within `<head>` section of an HTML file.
+
+•	**Inline CSS:** An inline style may be used to apply a unique style for a single element.  
```

Comment:
Use the proper unordered list in Markdown.
No need of unnecessary empty lines between list items.